tests/snapshots/test_selenium_chrome.test_selenium_chrome_pytest_rum_enabled.json

```diff
@@ -0,0 +1,182 @@
+         "test_session_id": "13460371773830286473",
+         "type": "test_module_end"
+       },
+       "metrics": {
+         "_dd.py.partial_flush": 1,
+         "_dd.tracer_kr": 1.0,
+         "_sampling_priority_v1": 1
+       },
+       "duration": 508369447,
+       "start": 1733222155431666123
+     },
+        {
+          "name": "pytest.test_suite",
+          "service": "pytest",
+          "resource": "pytest.test_suite",
+          "trace_id": 0,
+          "span_id": 3,
+          "parent_id": 2,
+          "type": "test",
+          "error": 0,
+          "meta": {
+            "_dd.base_service": "",
+            "_dd.origin": "ciapp-test",
+            "_dd.p.dm": "-0",
+            "_dd.p.tid": "674edf0b00000000",
+            "component": "pytest",
+            "language": "python",
+            "library_version": "2.18.0.dev93+g43e29a11b",
+            "os.architecture": "x86_64",
+            "os.platform": "Linux",
+            "os.version": "5.15.0-1071-aws",
+            "runtime.name": "CPython",
+            "runtime.version": "3.10.14",
+            "span.kind": "test",
+            "test.command": "pytest --ddtrace -s",
+            "test.framework": "pytest",
+            "test.framework_version": "8.3.4",
+            "test.module": "",
+            "test.module_path": "",
+            "test.status": "pass",
+            "test.suite": "test_selenium.py",
+            "test_module_id": "10039980247882256389",
+            "test_session_id": "13460371773830286473",
+            "test_suite_id": "2062522935705793595",
+            "type": "test_suite_end"
+          },
+          "metrics": {
+            "_dd.py.partial_flush": 1,
+            "_dd.tracer_kr": 1.0,
+            "_sampling_priority_v1": 1
+          },
+          "duration": 490880684,
+          "start": 1733222155431809947
+        }],
+[
+  {
+    "name": "pytest.test",
+    "service": "pytest",
+    "resource": "test_selenium.py::test_selenium_local_pass",
+    "trace_id": 1,
+    "span_id": 1,
+    "parent_id": 0,
+    "type": "test",
+    "error": 0,
+    "meta": {
+      "_dd.base_service": "",
+      "_dd.origin": "ciapp-test",
+      "_dd.p.dm": "-0",
+      "_dd.p.tid": "674edf0b00000000",
+      "component": "pytest",
+      "language": "python",
+      "library_version": "2.18.0.dev93+g43e29a11b",
+      "os.architecture": "x86_64",
+      "os.platform": "Linux",
+      "os.version": "5.15.0-1071-aws",
+      "runtime-id": "308c158b5dd547b19cea30ec737d97b4",
+      "runtime.name": "CPython",
+      "runtime.version": "3.10.14",
+      "span.kind": "test",
+      "test.command": "pytest --ddtrace -s",
+      "test.framework": "pytest",
+      "test.framework_version": "8.3.4",
+      "test.module": "",
+      "test.module_path": "",
+      "test.name": "test_selenium_local_pass",
+      "test.source.file": "test_selenium.py",
+      "test.status": "pass",
+      "test.suite": "test_selenium.py",
+      "test.type": "test",
```

Comment:
Shouldn't `test.browser.driver` and friends be showing up here?

Reply:
Yep. See earlier comment re: draft PR.